### DISCUSSION TITLE
`vercel dev` inside a turborepo project fails to start

### DISCUSSION THREAD

**Discussion by bitttttten** [vercel/community#4676] [Help]
### Summary

We are using the `/api` folder for Serverless Functions inside our turborepo powered monorepo. It's a vite project inside `apps/home`. We want to access our Serverless Functions locally with `vercel dev`, although this script fails.

We try and run `vercel dev` inside this project and it fails:

```sh
$ pwd
/bitttttten/my-mono-repo/apps/home

$ npx vercel dev
Vercel CLI 32.5.0
Error: /bitttttten/my-mono-repo/apps/home/apps/home doesn't exist

$ npx vercel dev --debug
> [debug] [2023-10-31T10:42:06.081Z] Found config in file "/bitttttten/my-mono-repo/apps/home/vercel.json"
Vercel CLI 32.5.0
> [debug] [2023-10-31T10:42:06.084Z] user supplied known subcommand: "dev"
> [debug] [2023-10-31T10:42:06.188Z] Found ".git/config" - detected "/bitttttten/my-mono-repo" as repo root
> [debug] [2023-10-31T10:42:06.196Z] Spinner invoked (Retrieving project…) with a 1000ms delay
> [debug] [2023-10-31T10:42:06.199Z] #1 → GET https://api.vercel.com/teams/TEAMID
> [debug] [2023-10-31T10:42:06.200Z] #2 → GET https://api.vercel.com/v9/projects/PROJECTID?teamId=TEAMID
> [debug] [2023-10-31T10:42:06.754Z] #1 ← 200 OK: sfo1::********-********-******** [555ms]
> [debug] [2023-10-31T10:42:06.795Z] #2 ← 200 OK: sfo1::********-********-******** [595ms]
> [debug] [2023-10-31T10:42:06.798Z] Fetching Environment Variables of project ******** and target undefined
> [debug] [2023-10-31T10:42:06.799Z] #3 → GET https://api.vercel.com/v1/env/pull/********?source=vercel-cli%3Adev&teamId=TEAMID
> [debug] [2023-10-31T10:42:07.003Z] #3 ← 200 OK: sfo1::********-********-******** [203ms]
Error: /bitttttten/my-mono-repo/apps/home/apps/home doesn't exist
> [debug] [2023-10-31T10:42:07.010Z] Error: /bitttttten/my-mono-repo/apps/home/apps/home doesn't exist
    at DevServer._start (/bitttttten/my-mono-repo/node_modules/vercel/dist/index.js:186987:17)
    at DevServer.start (/bitttttten/my-mono-repo/node_modules/vercel/dist/index.js:186975:36)
    at dev (/bitttttten/my-mono-repo/node_modules/vercel/dist/index.js:187515:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main4 (/bitttttten/my-mono-repo/node_modules/vercel/dist/index.js:187678:12)
    at async main1
```

It seems to look for `apps/home/apps/home` which might make sense since on vercel it thinks the working directory is `apps/home` as we're in a monorepo. If I run the above command in the root of the project, well, we haven't set up the root of the project and so vercel asks:

```sh
$ pwd
/bitttttten/my-mono-repo

$ npx vercel dev --cwd apps/home
Vercel CLI 32.5.0
Error: /bitttttten/my-mono-repo/apps/home/apps/home doesn't exist

$ npx vercel dev 
Vercel CLI 32.5.0
❗️  Your Project was either deleted, transferred to a new Team, or you don’t have access to it anymore.
? Set up and develop “/bitttttten/my-mono-repo”? [Y/n]
```

And I don't think we can do that as it doesn't make sense to make a new project since the root of the monorepo has no build step or dev command 😅 

Any help is appreciated!

### Steps to Reproduce

In short you can imagine this:

npx create-turbo -e basic
mkdir apps/home
cd apps/home
npm init -y
mkdir api
touch api/handler.ts
echo "..some code for a basic handler.." > api/handler.ts
npx vercel dev
